### DISCUSSION TITLE
articles/why-I-made-my-own-personal-docs-site

### DISCUSSION THREAD

**Discussion by giscus** [yehezkielgunawan/yehezgun-new#11] [Article Post]
# articles/why-I-made-my-own-personal-docs-site

Yehezkiel Gunawan's Article Post

https://yehezgun.com/articles/why-I-made-my-own-personal-docs-site

**Comment by yehezkielgunawan** [comment]
### 👋 Hi there! Hope you're all good.
You can discuss about this topic here or share this readings if you like it 😁
